tests/utils.go

```diff
@@ -2792,6 +2793,48 @@ func NewConsoleExpecter(virtCli kubecli.KubevirtClient, vmi *v1.VirtualMachineIn
 	}, timeout, opts...)
 }
 
+// The method adds the expect.BSnd command to the exect.BExp expression.
+// It is done to make sure the match was found in the result of the expect.BSnd
+// command and not in a leftover the wasn't removed from the buffer.
+func ExpectBatchWithValidatedSend(expecter expect.Expecter, batch []expect.Batcher, timeout time.Duration) ([]expect.BatchRes, error) {
+	sendFlag := 0
```

Comment:
i think it should enforce that batch start with SND?

Reply:
That was my approach when I started working on it. But there are case the first `BExp` looks for a match in the startup output.

Reply:
Done, added it to the method.